util/testing.go

```diff
@@ -49,12 +49,16 @@ func (panicTesterImpl) Error(args ...interface{}) {
 	panic(fmt.Sprint(args...))
 }
 
+func (panicTesterImpl) Errorf(format string, args ...interface{}) {
+	panic(fmt.Sprint(args...))
+}
+
 func (pt panicTesterImpl) Fatal(args ...interface{}) {
 	pt.Error(args...)
 }
 
-func (panicTesterImpl) Fatalf(format string, args ...interface{}) {
-	panic(fmt.Sprintf(format, args...))
+func (pt panicTesterImpl) Fatalf(format string, args ...interface{}) {
+	pt.Errorf(format, args...)
```

Comment:
leaving this as the panic and making the `Errorf` call `panicf` seems better imo.